jsonargparse/_parameter_resolvers.py

```diff
@@ -838,12 +871,16 @@ def get_parameters(self) -> ParamList:
             self.component, self.parent, self.logger
         )
         self.replace_param_default_subclass_specs(params)
+        if unpack_typed_dict_kwargs(params):
+            kwargs_idx = -1
         if args_idx >= 0 or kwargs_idx >= 0:
             self.doc_params = doc_params
             with mro_context(self.parent):
                 args, kwargs = self.get_parameters_args_and_kwargs()
             params = replace_args_and_kwargs(params, args, kwargs)
         add_stub_types(stubs, params, self.component)
+        # in case a typed-dict kwarg typehint is inherited
+        unpack_typed_dict_kwargs(params)
```

Comment:
Does this happen in the unit tests or is it just in case? If it doesn't happen in the unit test, then please either add a test for it don't add this. If you don't know of a test to make this happen, then please remove. If it is kept, then also inside a `if kwargs_idx >= 0:`.

Reply:
You are correct; I think the code is structured such that any inherited `Unpack` would already be resolved by this point. I will remove this check